client/my-sites/plan-features/item.jsx

```diff
@@ -26,8 +27,10 @@ export default function PlanFeaturesItem( { description, children } ) {
 		);
 	};
 
+	const classes = classNames( 'plan-features__item', className );
+
 	return (
-		<li className="plan-features__item">
+		<li className={ classes }>
```

Comment:
I think we shouldn't use `li` without enclosing `ul`. 

![selection_022](https://cloud.githubusercontent.com/assets/4988512/16657800/0eeadd8e-4464-11e6-9ceb-05e6eed6d46b.jpg)

http://stackoverflow.com/questions/2031725/is-using-li-without-enclosing-ul-tags-dangerous